src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -2054,6 +2054,10 @@ public object Convert(object[] values, Type targetType, object parameter, Cultur
 
             var textBlock = values[0] as TextBlock;
             var viewModel = values[1] as SearchViewModel;
+
+            if(viewModel == null)
```

Comment:
Here I need to add this check. Because I used new member template in `ClassInformationView` and its' ViewModel is not `SearchViewModel`.